packages/desktop-client/src/components/accounts/TransactionsTable.js

```diff
@@ -1704,23 +1704,25 @@ export let TransactionTable = React.forwardRef((props, ref) => {
         let { data, diff } = splitTransaction(newTrans, id);
         setNewTransactions(data);
 
-        // TODO: what is this for???
```

Comment:
Well.. now we know why this was necessary :)

Reply:
Well done. :D